portfolio/src/main/webapp/script.js

```diff
@@ -73,7 +73,15 @@ function getComments() {
   fetch(url) 
   .then(response => response.json())
   .then((messages) => {
-    document.getElementById('message-container').innerText = messages.join('\n');
+    document.getElementById('message-container').innerHTML = '';
+
+    for (message of messages) {
+      let commentContainer = document.createElement('div');
```

Comment:
this can be const. you are still allowed to modify const variables, you just can't reassign them.